Integrations/FidelisElevateNetwork/FidelisElevateNetwork-SS.py

```diff
@@ -176,6 +177,341 @@ def generate_time_settings(time_frame=None, start_time=None, end_time=None):
 
 ''' COMMANDS + REQUESTS FUNCTIONS '''
 
+##Code added by Dave for additional functionality - Second set of items
+
+#This section will change the explicit score of an alert - works
+def update_alertstatus_command():
+    args = demisto.args()
+    alert_id = args['alert_id']
+    explicitScore = args['explicitScore']
+
+    # FALSE POSITIVE    - explicitScore = 1
+    # NOT INTERESTING   - explicitScore = 2
+    # INTERESTING       - explicitScore = 3
+    # ACTIONABLE        - explicitScore = 4
+
+    data = {
+        'alertIds': [alert_id],
+        'explicitScore': explicitScore
+    }
+
+    result = update_alertstatus(alert_id, data)
+
+@logger
+def update_alertstatus(alert_id, data):
+    #url = '/j/rest/v1/alert/feedback/{}/'.format(alert_id)
+    url = '/j/rest/v1/alert/feedback/'
+    demisto.log(url)
+    demisto.log(str(data))
+    result = http_request('PUT',url, data=data)
+#end of explicit score section
+
+#grabs the decoding path data - works
+def get_alert_dpath_command():
+    args = demisto.args()
+    alert_id = args['alert_id']
+
+    result = get_alert_dpath(alert_id)
+
+    output = {
+        'ID': alert_id,
+        'dpath': result
+     }
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': result,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('Alert {}'.format(alert_id), result, headerTransform=pascalToSpace,
+                                         removeNull=True),
+        'EntryContext': {
+            'Fidelis.Alert(val.ID && val.ID == obj.ID)': output,
+        },
+    })
+
+@logger
+def get_alert_dpath(alert_id):
+    result = http_request('GET', '/j/rest/v1/alert/dpath/{}/'.format(alert_id))
+
+    return result
+# End of decoding path data code block
+
+#Forces a file to get submited to the sandbox for analysis - works
+def alert_ef_submission_command():
+    args = demisto.args()
+    alert_id = args['alert_id']
+
+    result = alert_ef_submission(alert_id)
+
+    output = {
+        'ID': alert_id,
+        'sessiondata': result
+     }
+
+    demisto.results({
```

Comment:
replace `demisto.results` with `return_outputs`
```python
return_outputs(
    readable_output=tableToMarkdown('Alert {}'.format(alert_id), result, headerTransform=pascalToSpace, removeNull=True),
    outputs={
        'Fidelis.Alert(val.ID && val.ID == obj.ID)': output,
    },
    raw_response=result
)
```
change in all the places accordingly

Reply:
code conventions
https://github.com/demisto/content/tree/master/docs/code_conventions